.github/config.yml

```
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
```

```suggestion
  Thanks for your contribution to Meshery! :tada:
```
And, 
We recommend using uniform name conventions for images throughout the project, update the names to
Meshery-celebration-Dark.svg
Meshery-celebration-Light.svg